testsuite/forge/src/backend/k8s/mod.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{Factory, Swarm};
+use tokio::runtime::Runtime;
+
+pub mod node;
+pub mod swarm;
+pub use swarm::*;
+
+pub struct K8sFactory {}
+
+impl K8sFactory {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl Factory for K8sFactory {
+    fn launch_swarm(&self, _node_num: usize) -> Box<dyn Swarm> {
+        let rt = Runtime::new().unwrap();
+        let swarm = rt.block_on(K8sSwarm::new()).unwrap();
+        Box::new(swarm)
```

Comment:
We should create a single runtime per K8sSwarm and store it inside of the K8sSwarm so that it can be easily reused instead of requiring use to create a new runtime each time we call an async fn.